mlir/lib/Dialect/Vector/Transforms/VectorTransforms.cpp

```diff
@@ -1300,7 +1300,7 @@ class DropInnerMostUnitDimsTransferRead
     if (dimsToDrop == 0)
       return failure();
 
-    auto inBounds = readOp.getInBoundsValues();
+    auto inBounds = readOp.getInBounds();
     auto droppedInBounds = ArrayRef<bool>(inBounds).take_back(dimsToDrop);
```

Comment:
nit: now that `getInBounds()` returns an ArrayRef this can just be:
```c++
    auto droppedInBounds = readOp.getInBounds().take_back(dimsToDrop);
```